cypress/integration/homework/classes/HomePage.js

```diff
@@ -0,0 +1,70 @@
+class HomePage {
+
+
+    clearButton = "#clearButton";
+    actionDropDown = "#selectOperationDropdown";
+    calculateButton = "#calculateButton";
+    numberField1 = "#number1Field";
+    numberField2 = "#number2Field";
+    integersOnlyCheckbox = "#integerSelect";
+
+
+    checkIfFieldsEnabled(field) {
+        cy.get(field).should('be.enabled')
+    }
+
+    enterFieldsWith(number1, number2, action) {
+        cy.get(this.numberField1).clear()
+        cy.get(this.numberField1).type(number1)
+        cy.get(this.numberField2).clear()
+        cy.get(this.numberField2).type(number2)
+        cy.get(this.actionDropDown).select(action)
+        cy.get(this.calculateButton).click()
+    }
+    checkAnswer(answer) {
+        cy.get('input[name="numberAnswer"]').invoke('val').should('eq', answer)
```

Comment:
* What about moving this _input selector_ to variable? :)
* You can use cypress comparison for the value. Please check here: https://docs.cypress.io/api/commands/should.html#Value
* Good practice is to use field id if it exists ;) Please check it.
(same for 30th line)